From 9b3031dcd2c020d7abfdea510fbea55a59fe5d3e Mon Sep 17 00:00:00 2001
From: Robby Cai <R63905@freescale.com>
Date: Wed, 10 Jul 2013 20:46:47 +0800
Subject: [PATCH 2628/2666] ENGR00263553 fb: Change type uint to unsigned int
 in struct mxcfb_update_data

This file could be included in user-space application while uint is not quite
standard type. This patch changed to unsigned int to avoid to include specific
header file. This is requested from mm team.

Signed-off-by: Robby Cai <R63905@freescale.com>
(cherry picked from commit 318a325b8c19d1ba7c1451797da5a161e787c64d)
---
 include/linux/mxcfb.h |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/include/linux/mxcfb.h b/include/linux/mxcfb.h
index e987efb..d96d1b7 100644
--- a/include/linux/mxcfb.h
+++ b/include/linux/mxcfb.h
@@ -110,7 +110,7 @@ struct mxcfb_update_data {
 	__u32 update_mode;
 	__u32 update_marker;
 	int temp;
-	uint flags;
+	unsigned int flags;
 	struct mxcfb_alt_buffer_data alt_buffer_data;
 };
 
-- 
1.7.7.4

